### PR TITLE
Feature: Upload image to cloudinary API and style page

### PR DESCRIPTION
**Pull request by chelsabeth**
In this PR I fixed my code to be much cleaner and more readable. The image is now uploading to the cloudinary API. I also styled the profile page to fit our wireframe

Trello: https://trello.com/c/tqLZJ9ny

### CODE REVIEW

File: src/components/pages/Profile/Profile.js [hunk 1]
```diff
@@ -1,76 +1,132 @@
...
 
-  const handleSubmitFile = e => {
-    e.preventDefault();
-    if (!previewSource) return;
-    uploadImage(previewSource);
-  };
+  display: flex;
+`;
+
+const Footer = styled.div`
```
Comment (vtopham) [inline src/components/pages/Profile/Profile.js:35]: Are we making a universal footer?

**Review by vtopham — APPROVED** [review]
Looks great! I saw something about a footer here - I'm making a note to talk about a global footer tomorrow morning :)

**Review by locbgiang — APPROVED** [review]
Looks good and clean.

**Comment by chelsabeth** [comment]
[Component: Profile](https://trello.com/c/tqLZJ9ny/47-profile)

**Pull request merged**